weave-js/src/components/PagePanelComponents/Home/Browse2/RunsTable.tsx

```diff
@@ -40,7 +45,8 @@ export type DataGridColumnGroupingModel = Exclude<
 function addToTree(
   node: GridColumnGroup,
   fields: string[],
-  fullPath: string
+  fullPath: string,
```

Comment:
The changes to this function are to fix a problem I saw when testing with Ken's project. He had outputs like `output.scores.score.rouge1` and `output.score.route1.mean` so `score` was being repeated as a groupId, which mui doesn't allow.